CONTRIBUTING.md

```diff
@@ -25,11 +25,12 @@ following installed:
 
 * [Python](https://www.python.org/downloads/)
     - If you're on Windows (Not Windows Subsystem for Linux)
-        - Install Python 2.7 x64
-        - Install at least one Python 3.x x64 - 3.4, 3.5 or 3.6 will all work
+        - Install at least one Python 3.x x64 - anything between 3.4 and 3.8 should work
```

Comment:
Let's not recommend EOLed versions
https://devguide.python.org/#status-of-python-branches
```
        - Install at least one Python 3.x x64 - anything higher than 3.6 should work
```